drivers/rpmsg/Kconfig

```diff
@@ -15,6 +15,18 @@ config RPMSG_CHAR
 	  in /dev. They make it possible for user-space programs to send and
 	  receive rpmsg packets.
 
+config RPMSG_SYSLOG
+	tristate "RPMSG syslog redirection"
+	depends on RPMSG
+	help
+	  Say Y here to redirect the syslog/printf from remote processor into
+	  the kernel log which is very useful to see what happen in the remote
+	  side.
+
```

Comment:
Not sure I see the benefit from your explanations. if there is already an access to the syslog, pushing it into the kernel log will add more noise. Unless this helps correlate between activity on the remote processor and the main host I don't see the value.